mdpow/tests/test_forcefields.py

```diff
@@ -152,22 +152,6 @@ def test_get_solvent_wetoctanol(forcefield):
         assert (mdpow.forcefields.get_solvent_model(model, forcefield=forcefield) is
                 mdpow.forcefields.GROMACS_SOLVENT_MODELS[forcefield][model])
 
-    @staticmethod
```

Comment:
Why were the tests removed?

Reply:
These tests do the same thing as the previous 3 tests.